src/stream/stream/timeout.rs

```diff
@@ -0,0 +1,59 @@
+use std::error::Error;
+use std::fmt;
+use std::pin::Pin;
+use std::time::Duration;
+
+use futures_timer::Delay;
+
+use crate::future::Future;
+use crate::stream::Stream;
+use crate::task::{Context, Poll};
+
+#[doc(hidden)]
```

Comment:
Remove `#[doc(hidden)]`, and add a doc comment linking back to `Stream::timeout`.